sdk/ai/azure-ai-generative/azure/ai/generative/synthetic/simulator/_model_tools/models.py

```diff
@@ -51,51 +58,52 @@ def __init__(self, n_retry, retry_timeout, logger, retry_options=None):
         trace_config.on_request_end.append(self.on_request_end)
         if retry_options is None:
             retry_options = RandomRetry(  # set up retry configuration
-                statuses=[104, 408, 409, 424, 429, 500, 502,
-                        503, 504],  # on which statuses to retry
+                statuses=[104, 408, 409, 424, 429, 500, 502, 503, 504],  # on which statuses to retry
                 attempts=n_retry,
                 min_timeout=retry_timeout,
                 max_timeout=retry_timeout,
             )
 
-        self.client = RetryClient(
-            trace_configs=[trace_config], retry_options=retry_options)
+        self.client = RetryClient(trace_configs=[trace_config], retry_options=retry_options)
 
-    async def on_request_start(self, session, trace_config_ctx, params):
```

Comment:
Why remove `sessions` param? Was it unused?

Reply:
Yes, it is unused. Should I add it back?